Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -206,11 +212,25 @@ public IEnumerator NearInteractionTouchableVolumeVariant()
 
             using (var catcher = CreateEventCatcher(touchable))
             {
-                // Touch started and completed when entering and exiting the collider
+                // Touch started when entering collider
                 yield return PlayModeTestUtilities.MoveHandFromTo(initialHandPosition, objectPosition, numSteps, ArticulatedHandPose.GestureId.Open, Handedness.Right, inputSim);
                 Assert.AreEqual(1, catcher.EventsStarted);
                 Assert.AreEqual(0, catcher.EventsCompleted);
-                yield return PlayModeTestUtilities.MoveHandFromTo(objectPosition, rightPosition, numSteps, ArticulatedHandPose.GestureId.Pinch, Handedness.Right, inputSim);
+
+                // No touch up when moving through collider
+                Vector3[] cornerPositions = new Vector3[8];
```

Comment:
nit: Elaborate a bit more with something like "Ensure no touch up event fires while moving hand/pokepointer through collider to each corner of volume"